examples/mqtt-client-ssl/mqtt_client_ssl.md

```diff
@@ -0,0 +1,23 @@
+# MQTT client - SSL/TLS
+
+The `mqtt:Client` connects to a MQTT server via SSL/TLS and then, sends messages to the server. SSL/TLS can be enabled by configuring the `secureSocket`, which requires a certificate or a truststore. Further, mTLS can be enabled by providing a certificate and private key of the client or a keystore. Use this to connect to a MQTT server secured with SSL.
```

Comment:
Did we consider adding a separate mTLS BBE for MQTT ? other modules have separate BBE for mTLS

Reply:
Messaging modules only have TLS BBEs.